### DISCUSSION TITLE
name of the drawing surface class for the Android Joplin plugin?

### DISCUSSION THREAD

**Discussion by jdkruzr** [personalizedrefrigerator/js-draw#81] [General]
hey there -- per [this guide](https://gist.github.com/calliecameron/b3c62c601d255630468bd493380e3b7e), I'm trying to activate  Boox's support for "instant" e-ink drawing but I cannot find the name of the class to put in the eac_config file. I suspect it's not as straightforward as it would usually be because js-draw is being used by some component of the Kotlin software that comprises the Android Joplin app. can you point me in the right direction? thanks.

**Comment by personalizedrefrigerator** [comment]
Hi,
`js-draw` is being used within a WebView in Joplin. The WebView is created here: https://github.com/laurent22/joplin/blob/26ae3f853e92bf4ddfd2c82aabf6780e43cf685c/packages/app-mobile/components/NoteEditor/ImageEditor/ImageEditor.tsx#L348 , but it's a React Native component and not a native Android View.

If there's a way to enable e-ink drawing for individual webpages, it's more likely to be applicable to `js-draw`.

**Reply by jdkruzr** [reply]
thanks. what does that mean for where to look?

**Reply by personalizedrefrigerator** [reply]
Does `com.reactnativecommunity.webview.RNCWebView` work? (From [this dependency of Joplin](https://github.com/react-native-webview/react-native-webview/blob/master/android/src/main/java/com/reactnativecommunity/webview/RNCWebView.java)). Note that this would also apply to Joplin's note editor and viewer.

**Reply by jdkruzr** [reply]
yes it does! thanks so much!
